src/applications/coronavirus-screener/components/FormResult.unit.spec.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { expect } from 'chai';
+import FormResult from './FormResult';
+
+describe('FormResult', () => {
+  it('renders FormResult component', async () => {
+    const formState = {
+      status: 'incomplete',
+    };
+    render(<FormResult formState={formState} />);
+
+    // screen.debug();
+
+    expect(await screen.findByText('Please answer all the questions above.')).to
+      .not.be.null;
```

Comment:
`findByText` will throw an error if it doesn't find that text. The `expect` is sort of redundant. The cheatsheet is super useful, https://testing-library.com/docs/react-testing-library/cheatsheet.

```suggestion
    await screen.findByText('Please answer all the questions above.');
```

Reply:
how does the shorthand behave in a context where there are multiple `expect()` statements?

Reply:
I'm not sure I follow. This test case will stop until the `await` resolves, in which case it'll either throw an error or do nothing (since nothing is assigned to the result of the `findByText`)

Reply:
This was just a nitpick anyway but I personally love how succinct RTL can make a test

Reply:
@ncksllvn i was wondering in the case where there is a test with 5 expects, wanting to know that 3rd one failed but following 2 passed instead of erroring out after 3rd one. anyway that's not occurring here so i'm going with shorthand, thanks!